backend/tests/Feature/Sync/CollectionSnapshotSessionAuthTest.php

```
@@ -0,0 +1,46 @@
+ * cookie every Inertia page uses — so it works on any host / behind any reverse
+ * proxy, without depending on Sanctum stateful-domain matching. Regression guard
+ * for #193 ("data saved but lists show nothing"): the snapshot 401'd on hosts
+ * APP_URL didn't cover, and the frontend swallowed the error into an empty list.
+ */
+class CollectionSnapshotSessionAuthTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function test_authenticated_session_gets_the_snapshot(): void
+    {
+        $user = User::factory()->create();
```

PHPStan (2.2.5)

[error] Call to an undefined static method App\Models\User::factory().


(staticMethod.notFound)